### DISCUSSION TITLE
How do I build libSkiaSharp and libHarfBuzzSharp into static libraries? ( *.a for Linux/macOS and *.lib for Windows because Net 7.0 and 8.0 has NativeAot support with DirectInvoke?

### DISCUSSION THREAD

**Discussion by DeafMan1983** [mono/SkiaSharp#2475] [Q&A]
Hello everyone, how do I build libraries into static if I use AvaloniaAOT with both native libraries *.a via clang after `Generating native code`

Thanks!

**Comment by aakrandan** [comment]
@DeafMan1983 did you find how to fix this issue?

**Comment by DeafMan1983** [comment]
No fixes yet! I think I will find other way replace with Skia

**Comment by aakrandan** [comment]
But if we build using skia, then we wont be able to use skiasharp APIs right?
I created libskia.a static library using skia, but i need to create it one for skiasharp

**Comment by DeafMan1983** [comment]
Because compilation got errors and incompatible that's why we need change to blend2d Because blend2d works as well for latest build-essential.

**Comment by aakrandan** [comment]
I don't know if this helps, but if you are natively developing your application for macOS using Swift you can use this 
https://github.com/migueldeicaza/SkiaKit

It is still under development, but with help i guess we can get it on par with the SkiaSharp implementation.
